src/lib.rs

```diff
@@ -8,3 +8,8 @@ mod flatfiledirmgr;
 mod http;
 
 pub use client::Client;
+pub use client::AUTHORITY_FILENAME;
```

Comment:
don't like to repeat myself w/ variables like that, so I exposed them

not sure if that's the "right way"?